### ISSUE TITLE
Which hidden layer has to be chosen as latent vector for passing to Loss function?

### ISSUE THREAD

**Issue by Nikronic** [Nikronic/DetailsNet#3]
Which hidden layer should be chosen as a low-spatial latent vector to pass to loss function?
In the DetailsNet, it seems layers 2 and 5 have been chosen on VGG19BN.

Note: the last layer of VGGs is classifier so we cannot use them because of 1-D vector representation of the last layer to match with a class of object.

**Comment by Nikronic** [comment]
Combination of layer 2 and 5. What kind of combination? No information!
![image](https://user-images.githubusercontent.com/32401852/58370777-bcd06b00-7f1f-11e9-8f64-8072fd7e2010.png)


- [x] Check tensorflow or other libs implementation to figure this out.

**Comment by Nikronic** [comment]
We use pool2 and pool5 outputs as feature and combine them by non-weighted adding of MSE_LOSSes.
